Assets/SEE/Game/Worlds/PlayerSpawner.cs

```diff
@@ -9,37 +11,108 @@
+        /// <exception cref="Exception">thrown in case the <paramref name="prefabName"/> cannot be loaded</exception>
+        public SpawnInfo(string prefabName, Vector3 position, int rotation)
         {
-            [Tooltip("Avatar game object used as prefab.")]
-            public GameObject PlayerPrefab;
+            PlayerPrefab = Resources.Load<GameObject>(Path(prefabName));
+            if (PlayerPrefab == null)
+            {
+                throw new Exception($"Player prefab {Path(prefabName)} not found.\n");
+            }
+
+            Position = position;
+            Rotation = rotation;
 
-            [Tooltip("World-space position at which to spawn.")]
-            public Vector3 Position;
+            return;
 
-            [Tooltip("Rotation in degree along the y axis.")]
-            public float Rotation;
+            static string Path(string prefabName) => $"{playerPrefabFolder}/{prefabName}";
         }
+    }
 
+    /// <summary>
+    /// Spawns players in multi-player mode.
+    /// </summary>
+    public class PlayerSpawner : NetworkBehaviour
+    {
         /// <summary>
         /// The information needed to spawn player avatars.
         /// </summary>
+        /// <remarks>This field must not be readonly. It will be changed by Odin during serialization.</remarks>
         [Tooltip("The information to be used to spawn players."), ShowInInspector, SerializeField]
-        private List<SpawnInfo> playerSpawns = new();
+        private List<SpawnInfo> playerSpawns;
+
+        /// <summary>
+        /// The name of the player prefabs used for spawning. These prefabs must be located in the
+        /// <see cref="SpawnInfo.playerPrefabFolder"/>.
+        /// </summary>
+        public static List<string> Prefabs = new()
+        {
+           "Male1",
+           "Male2",
+           "Male3",
+           "Female1",
+           "Female2",
+           "Female3",
+        };
 
         /// <summary>
         /// The dissonance communication. Its game object holds the remote players as its children.
         /// </summary>
         private DissonanceComms dissonanceComms = null;
 
+        /// <summary>
+        /// Initializes the player spawns if they are not already set by the user in the Unity inspector.
+        /// </summary>
+        private void Awake()
+        {
+            if (playerSpawns == null || playerSpawns.Count == 0)
+            {
+                playerSpawns = new()
+                {
+                    new SpawnInfo(Prefabs[0], new Vector3(0.4f, 0f, -5.8f), 270),
+                    new SpawnInfo(Prefabs[1], new Vector3(0.4f, 0f, -6.6f), 270),
+                    new SpawnInfo(Prefabs[2], new Vector3(0.4f, 0f, -7.8f), 270),
+                    new SpawnInfo(Prefabs[3], new Vector3(0.4f, 0f, -5.8f), 270),
+                    new SpawnInfo(Prefabs[4], new Vector3(0.4f, 0f, -6.8f), 270),
+                    new SpawnInfo(Prefabs[5], new Vector3(0.4f, 0f, -7.8f), 270),
```

Comment:
If this used for spawn positions, why do the first three prefabs have a different Z coordinate, but the latter three match the first three? Shouldn't all of them be different from each other?

Reply:
The female avatars are now on the opposite side facing the male avatars.